packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -100,10 +100,13 @@ describe( 'Multi-entity save flow', () => {
 
 			// Add a template part and edit it.
 			await insertBlock( 'Template Part' );
+
+			await page.waitForXPath( createNewButtonSelector );
 			const [ createNewButton ] = await page.$x(
 				createNewButtonSelector
```

Comment:
Let's try updating this selector to use a hardcoded test id or something. If that improves stability then we know my theory is might be correct.